src/external/tinygltf/tiny_gltf.h

```diff
@@ -4,7 +4,7 @@
 //
 // The MIT License (MIT)
 //
-// Copyright (c) 2015 - 2017 Syoyo Fujita, Aurélien Chatelain and many
+// Copyright (c) 2015 - 2018 Syoyo Fujita, Aurélien Chatelain and many
```

Comment:
Can you amend the commit message that updates this file and say which upstream SHA-1 is this? Helps a lot when updating later.

Thanks.